src/main/java/seedu/commands/FindCommand.java

```diff
@@ -111,20 +116,24 @@ private <T extends EntryType> ArrayList<T> getMatchingAmount(String findValue, A
         double lower;
         double upper;
         if (findValue.contains(FIND_RANGE_DIVIDER)) { // range
-            String[] range = findValue.split(FIND_RANGE_DIVIDER);
-            lower = CommandUtils.formatAmount(range[0], FIND_COMMAND_FORMAT);
-            upper = CommandUtils.formatAmount(range[1], FIND_COMMAND_FORMAT);
+            String[] rangeValues = extractRangeValues(findValue);
+            lower = CommandUtils.formatAmount(rangeValues[0], FIND_COMMAND_FORMAT);
+            upper = CommandUtils.formatAmount(rangeValues[1], FIND_COMMAND_FORMAT);
```

Comment:
should we refractor the 0s and 1s in the code

Reply:
ok will do in the next commit